### ISSUE TITLE
Accessibility: Test And Report

### ISSUE THREAD

**Issue by wordshaker** [DDDEastMidlandsLimited/dddem-web#50]
### Describe the feature

We would like the DDD East Midlands website ran through accessibility testers and for issues to be raised based on the improvements that are made apparent by these tools

It would be great if we could meet the recommendations on the [a11y Web Accessibility Checklist](https://a11yproject.com/checklist)

Options for tools include:

* [pa11y](http://pa11y.org/) 
* [Lighthouse](https://developers.google.com/web/tools/lighthouse/)

But we don't mind what tools are used (not our area of expertise)

### Advantages

* To make the site more accessible to a wider audience

#### Acceptance Criteria

- [ ] A tool has been used to highlight any accessibility fixes needed on the site 

- [ ] Issues are raised for potential fixes/enhancements are are tagged with the `accessibility` label

**Comment by shzwlkr** [comment]
I can help with this 👍

**Comment by shzwlkr** [comment]
Initial Lighthouse audit results: 
<img width="631" alt="Lighthouse_audit" src="https://user-images.githubusercontent.com/22634756/66320696-3c7f3500-e917-11e9-8bf3-b9a32859f780.PNG">

Will create some issues based on the things highlighted in there.

**Comment by aitchiss** [comment]
I have raised an issue flagged in accessibility audit tools - #99 - and opened a pull request with relevant fixes. This complements the alt attributes PR already opened and referenced here

**Comment by redahaq** [comment]
Hi @wordshaker, is this issue something I can help with? I would love to make my first open source contribution!

- I can see that the site is missing the lang attribute for which I'll raise an issue and then PR.
![image](https://user-images.githubusercontent.com/39345463/71756771-7e256a80-2e89-11ea-9bb6-77e4dd368e94.png)

- Not a11y related, but would you also like help with SEO? I'd be happy to raise enhancement issues if so. Have just looked over the contribution guidelines :)
![image](https://user-images.githubusercontent.com/39345463/71756797-a90fbe80-2e89-11ea-9f12-6ed6d4290df2.png)
